x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_executed_details.test.tsx

```diff
@@ -51,85 +53,249 @@ describe('AuditExecutedDetails', () => {
       const wrapper = mountWithIntl(
         <TestProviders>
           <AuditdExecutedCommandLine
-            id={'hello-i-am-an-id'}
-            hostName={'host-1'}
-            userName={'username-1'}
-            processName={'process-1'}
-            processTitle={'process-title-1'}
-            workingDirectory={'working-directory-1'}
-            args={'arg1 arge 2 arg3'}
+            id="hello-i-am-an-id"
+            hostName="host-1"
+            userName="username-1"
+            processName="process-1"
+            processTitle="process-title-1"
+            workingDirectory="working-directory-1"
+            args="arg1 arge 2 arg3"
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual(
+        'Sessionusername-1@host-1inworking-directory-1executedprocess-1arg1 arge 2 arg3'
+      );
+    });
+
+    test('it returns a session with username if username, primary, and secondary all equal each other ', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <AuditdExecutedCommandLine
+            id="hello-i-am-an-id"
+            hostName="host-1"
+            userName="username-1"
+            primary="username-1"
+            secondary="username-1"
+            processName="process-1"
+            processTitle="process-title-1"
+            workingDirectory="working-directory-1"
+            args="arg1 arge 2 arg3"
```

Comment:
> is `arge 2` intentional?
same question here (and the remaining tests in this file)

Reply:
There's quite a few. I will get them all.